scene/gui/rich_text_label.cpp

```diff
@@ -1968,6 +1982,7 @@ RichTextLabel::RichTextLabel() {
 	tab_size = 4;
 	default_align = ALIGN_LEFT;
 	underline_meta = true;
+	override_selected_font_color = true;
```

Comment:
so changing this to false to make it the default would basically make it always disabled except if someone really wants to enable it...